doc-extraction/va-forms-system-core.api.md

```diff
@@ -3,31 +3,17 @@
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
+
 /// <reference types="react" />
 
-import { Context } from 'react';
 import { FieldHookConfig } from 'formik';
-
-// @beta
-export interface ChapterProps {
-  // (undocumented)
-  children: typeof Page[];
-  // (undocumented)
-  path: string;
-  // (undocumented)
-  title: string;
-}
+import { default as React_2 } from 'react';
 
 // Warning: (ae-forgotten-export) The symbol "CheckboxProps" needs to be exported by the entry point index.d.ts
 //
 // @public (undocumented)
 export const CheckboxField: (props: CheckboxProps) => JSX.Element;
 
-// Warning: (ae-forgotten-export) The symbol "CheckboxGroupProps" needs to be exported by the entry point index.d.ts
-//
-// @public (undocumented)
-export const CheckboxFieldGroup: (props: CheckboxGroupProps) => JSX.Element;
```

Comment:
I looked around in the PR and can't seem to find where this gets added back. Can we double check and make sure the docs got generated right?

Reply:
Checked for this, it's in my current version, so it'll be there when I push it up